tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27, py37
+isolated_build = True
+
+[testenv]
+deps =
+  cookiecutter
+  flake8
+  pytest
```

Comment:
BeeWare also uses the `pytest-tldr` plugin to make the `pytest` output more consumable (especially on test failure)